chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -86,6 +86,13 @@ def raise_argument_errors(self):
                     multiclass dataset types, their outputs are not confidences and are not \
                     compatible with any implemented calibration methods for classification."
             )
+        if self.uncertainty_method == "dirichlet":
+            raise NotImplementedError(
```

Comment:
This should not be a `NotImplementedError`, but something more like a custom `InvalidConfigurationError` or I suggest just a plain old `RuntimeError`. See the [Python docs](https://docs.python.org/3/library/exceptions.html#NotImplementedError) - `NotImplementedError` is only for base classes with methods that are required to be overridden.